index.html

```diff
@@ -454,7 +454,9 @@ <h3>sensitive</h3>
             </tr>
             <tr>
               <td>Domain:</td>
-              <td><code>Object</code></td>
+              <td>
+                <code>Object</code> | <code>Link</code>
```

Comment:
I did try, but I couldn't get xref to work with link these to the activitystreams-vocabulary document, I'm not sure why. I don't really understand xref, but we didn't use it before, so hopefully this remains clear enough  without it.